File: app/src/main/java/com/odc/ticket/data/models/MarkCompleteModel.java
package com.odc.ticket.data.models;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings({"unused","RedundantSuppression"})
public class MarkCompleteModel {

    @SerializedName("transferId")
    private long id;

    @SerializedName("completeNote")
    private String note;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }
}

File: app/src/main/java/com/odc/ticket/data/models/room/MarkCompleteModel.kt
package com.odc.ticket.data.models.room

import androidx.room.ColumnInfo
import androidx.room.PrimaryKey

data class MarkCompleteModel (
    @PrimaryKey(autoGenerate = true) val transferId: Int,
    @ColumnInfo(name = "completeNote") val completeNote: String,
)
